infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServerCache.java

```diff
@@ -185,6 +160,7 @@ public void invalidateAll() {
      */
     public void updateMaxConflictAddress(long newMaxConflictWildcard) {
         log.info("updateMaxConflictAddress, new address: {}", newMaxConflictWildcard);
+        invalidateUpTo(newMaxConflictWildcard);
         maxConflictWildcard = newMaxConflictWildcard;
         maxConflictNewSequencer = newMaxConflictWildcard;
```

Comment:
I think we can remove this variable, since holds redundant information. We only need maxConflictWildCard every time we boot strap the sequencer, we can set maxConflictWildCard = globalAddress(tail from bootstrap msg). 

@annym can further comment.

Reply:
Mmm perhaps we should keep them. Because imagine we have maxConflictNewSequencer = 60 (when we elected this sequencer as primary, that was the log's tail) and quickly maxConflictWildcard becomes 200 (because there was sequencer overflow) and we have a txSnapshot @ 50. Should the one @ 50 fail due to new sequencer or overflow? Also, we would need to remove the cause NEW_SEQUENCER cause we wouldn't be able to distinguish if the eviction was because cache invalidation or overflow.

Reply:
Yer. It is two different type of cache miss for different reasons.